### PR TITLE
Feat/auth

### PR DESCRIPTION
**Pull request by ajay-mandaviya**
Login Page
Signup Page
Screenshot from 2022-02-12 02-54-04
Screenshot from 2022-02-12 02-54-15

### CODE REVIEW

**Review by Isha2806 — APPROVED** [review]
Looks goood!

File: style/login.css [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  justify-content: center;
+  align-items: center;
+  cursor: pointer;
+}
+.check-account i {
+  margin-left: 0.5rem;
+}
+@media only screen and (max-width: 600px) {
+  .login-container {
+    width: 250px;
```
Comment (shivanigangadharan) [inline style/login.css:77]: Just a suggestion, can use clamp for login container width.
Eg : `width: clamp(250px, 40vw, 350px)`

**Review by shivanigangadharan — APPROVED** [review]
Code looks good to merge.

**Review by Bharatrathod-81 — APPROVED** [review]
perfect

**Pull request merged**